nx/exercises/exercises-1-20.livemd

```diff
@@ -0,0 +1,196 @@
+# Exercises: 1-20
+
+```elixir
+Mix.install([{:nx, github: "elixir-nx/nx", sparse: "nx"}])
+```
+
+## Introduction
+
+Inspired by the Python notebook _100 Numpy Exercises_:
+
+https://www.kaggle.com/code/utsav15/100-numpy-exercises/notebook
+
+## 1-10
+
+1. Install Nx in a Livebook (★☆☆)
+
+```elixir
+Mix.install([{:nx, github: "elixir-nx/nx", sparse: "nx"}])
+```
+
+2. Print the Nx version and the configuration (★☆☆)
+
+```elixir
+version = Application.spec(:nx, :vsn)
+# I'm not sure if there's an equivalent for "configuration"
+```
```

Comment:
I don't think this is really relevant in Elixir because we always have the package version scoped per project. In Python this is more of a problem due to global packages and virtual envs

Reply:
Agreed. I honestly don't even think printing the version is all that useful since you have to opt into a version at the top of the notebook. (Since, ya know, we have sane package management 😉)

This one is a good example of an exercise we may not want to include for whatever reason. Feel free to go through and mark or any others as "not a good candidate" or similar.

Reply:
Yes, I think we should not include this one

Reply:
[f9c005c](https://github.com/elixir-nx/nx/pull/1363/commits/f9c005cd5be16578574a3df51d0e2a348f379beb)